Add an entry for Apache Mesos

martin-g opened the pull request.

martin-g committed:
Add an entry for Apache Mesos

ZhengZhenyu commented:
could you update to use this logo?
https://en.wikipedia.org/wiki/Apache_Mesos#/media/File:Apache_Mesos_Logo.svg

martin-g committed:
Update the PNG logo with an SVG one

Requested-by: ZhengZhenyu
https://github.com/Linaro/ecosystemlandscape/pull/3#issuecomment-969940030

martin-g commented:
Done!
